### PR TITLE
Move all CSS for different color stops into 1 file

### PR DESCRIPTION
**Pull request by designedbyscience**
Instead of loading different CSS files for different stops, now all component CSS is scoped under a class on the body tag, which is applied at load based on the PS color stop.

### CODE REVIEW

File: src/index-build.html [hunk 1]
```diff
@@ -54,7 +55,7 @@
             }
         });
 
-        require(["js/init-build"]);
+        require(["js/init"]);
```
Comment (volfied) [inline src/index-build.html]: Do we need to fix the `paths` in require.config?

**Comment by volfied** [comment]
Thank you for making the less/CSS changes @designedbyscience, the PR needs a bit more work to get it working correctly for compile builds. But since you graciously offered your help with less, I'll take care of the rest.

**Comment by designedbyscience** [comment]
Ah sorry, didn't even look at the config...

**Comment by volfied** [comment]
@iwehrman Can you review our changes? I've gotten it to work under release as well. 

re@init-debug.js > init.js: These will be cleaned up nicer with webpack changes.

**Comment by iwehrman** [comment]
This currently has failing CI checks.

**Comment by iwehrman** [comment]
```
    Running "jshint:all" (jshint) task
       src/js/actions/layers.js
       1068 |            actionName = strings.ACTIONS.RENAME_LAYER,
                                      ^ 'strings' is not defined.
       1571 |            actionName = locked ? strings.ACTIONS.LOCK_LAYER : strings.ACTIONS.UNLOCK_LAYER,
                                               ^ 'strings' is not defined.
       1571 |            actionName = locked ? strings.ACTIONS.LOCK_LAYER : strings.ACTIONS.UNLOCK_LAYER,
                                                                            ^ 'strings' is not defined.
       2238 |                        name: strings.ACTIONS.CREATE_ARTBOARD
                                           ^ 'strings' is not defined.
```

**Comment by volfied** [comment]
https://github.com/adobe-photoshop/spaces-design/blob/designedbyscience/the-one-less-file/src/js/actions/layers.js#L1068

**Comment by volfied** [comment]
Travis is tripping.

File: src/index-build.html [hunk 2]
```diff
@@ -25,6 +25,7 @@
 <head>
     <meta charset="utf-8">
     <title>Photoshop Design Space (optimized build)</title>
+    <link rel="stylesheet/less" type="text/css" href="style/main.css" />
```
Comment (iwehrman) [inline src/index-build.html]: Is the `rel=stylesheet/less` part correct here?

**Comment by volfied** [comment]
@iwehrman Good catch, I had it changed locally on the built file but forgot it for the source file.

File: src/js/init.js [hunk 3]
```diff
@@ -29,11 +29,6 @@ define(function (require) {
     var ui = require("./util/ui"),
```
Comment (iwehrman) [inline src/js/init.js]: I find this renaming to be more confusing than it was before, but I guess it's all going away soon...

File: src/style/main.less [hunk 4]
```diff
@@ -281,3 +151,32 @@ input {
   html { font-size: @base-11; }
 
 }
+
+/* Setup Color Stops */
+.light {
```
Comment (iwehrman) [inline src/style/main.less]: I think all these class names need to be prefixed. Something like `.color-stop__light`, `color-stop__original`, etc.

**Comment by iwehrman** [comment]
Back to @volfied / @designedbyscience for the class-name prefixing. Looks good otherwise! Thanks for fixing my superhack!

**Comment by iwehrman** [comment]
Let's give it a whirl!

**Pull request merged**